library/core/src/pin.rs

```diff
@@ -793,6 +793,44 @@ impl<T: ?Sized> Pin<&'static T> {
     }
 }
 
+impl<'a, P: DerefMut> Pin<&'a mut Pin<P>> {
+    /// Gets a pinned mutable reference from this a nested pinned pointer.
```

Comment:
Seems like there's a typo in the second half of this sentence?

Reply:
Good catch! Fixed.